ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -37,7 +37,13 @@ class PhaserUnit extends Phaser.GameObjects.Container {
 
 		this.followListener = unit.on('follow', () => {
 			console.log('PhaserUnit follow', unit.id()); // TODO remove
-			scene.cameras.main.startFollow(this, true, 0.05, 0.05);
+			const camera = scene.cameras.main as Phaser.Cameras.Scene2D.Camera & {
```

Comment:
could you explain the syntax here? I obviously trust you with this, I am just confused

Reply:
I'm extending camera type with `_follow` which is private, but I need to use it to determine if camera is already following that entity. compiled code is much simpler:
https://github.com/moddio/taro/pull/324/files/26cdb18fb3e2dfb0a9681e5b970ab99120750654#diff-25cecd88e11153f0ecdf869e5e323ce73301c622fa4ebff0241dced268cc175aR31-R34